Bindings/Java/tests/TestEditMarkers.java

```diff
@@ -18,7 +18,7 @@ public static void main(String[] args) {
         Marker newMarker = new Marker();
         newMarker.setName(newMarkerName);
         newMarker.set_location(offset);
-        newMarker.setParentFrameName(body.getName());
+        newMarker.setParentFrame(body);
         context.cacheModelAndState();
```

Comment:
@chrisdembia If setting names is not supported anymore, this may have wide ranging effect since there are places in the  GUI that work with names/Strings and will need to change to use Components of specific type instead.

Reply:
I think the modelBuildingUtilties work with Strings/names and primitive types exclusively to avoid having users manipulating paths or Objects. Are these integrated into testing? @carmichaelong can you comment?

Reply:
Nice catch @aymanhab. ModelBuilder.py does use strings, but there's at least one location that must be updated as a result of sets being components: https://github.com/opensim-org/opensim-gui/blob/f9ad640d243aacdf825a8c2345fb7fa5c150ce48/Gui/opensim/Scripts/ModelBuilder.py#L32

That is not related to this PR; it is related to the previous PR about sets.